test/handle.js

```diff
@@ -29,10 +29,17 @@ describe('handle', () => {
     throw Boom.notFound()
   }
 
+  const badRequest = () => {
+    const err = Boom.badRequest()
+    err.output.payload.data = { foo: 'bar' }
```

Comment:
I believe you can pass the data to the constructor, like this:
```js
const badRequest = () => {
  throw Boom.badRequest({ foo: 'bar' })
}
```

Reply:
You can, but...
* It's the second argument `Boom.badRequest(message, data)`, which is annoying.
* [Hapi ignores the data argument](https://github.com/hapijs/boom#faq) suggesting to [modify the payload directly](https://github.com/hapijs/hapi/blob/master/API.md#error-transformation). I had mixed feelings about this but chose to do it this way to stay in the spirit of things.

Reply:
Double ugh.  Ugh _two times_.  Now I _still_ don't like Hapi.